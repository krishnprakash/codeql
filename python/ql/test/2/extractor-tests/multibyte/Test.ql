import python

from StrConst s
select s, s.getText()
