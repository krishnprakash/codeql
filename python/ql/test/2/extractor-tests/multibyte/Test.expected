| test.py:3:1:3:11 | Str | \u0111\u0142e\u00b6\u014b\u00b6\u0142\u014b |
| test.py:3:15:3:19 | Str | hi |
| test.py:3:23:3:27 | Str | \n |
| test.py:4:1:4:11 | Str | \u0111\u0142e\u00b6\u014b\u00b6\u0142\u014b |
| test.py:4:15:4:19 | Str | hi |
| test.py:4:23:4:27 | Str | \n |
